packages/devextreme/scss/widgets/fluent/list/_sizes.scss

```diff
@@ -13,6 +14,7 @@ $fluent-list-next-button-horizontal-padding: null !default;
 
 @if $size == "default" {
   $fluent-list-vertical-padding: 4px !default;
+  $fluent-list-bottom-padding: 4px !default;
```

Comment:
We shouldn't concrete values for 'default', 'compact' themes. $fluent-list-vertical-padding values too. Let's refactor this part.

Reply:
[640d4e0](https://github.com/DevExpress/DevExtreme/pull/27093/commits/640d4e0d7b58662795cac821b7a1415c4d807f6d)